src/lib.rs

```diff
@@ -1653,6 +1653,96 @@ impl Bindings {
             ))
         }
     }
+    
+    /// Write these bindings as source text to stdout.
+    pub fn write_to_stdout(&self) -> io::Result<()> {
+        let bindings = self.to_string();
+        
+        match self.rustfmt_generated_string(bindings) {
+            Ok(Some((rustfmt_bindings, rustfmt_stderr))) => { 
+                println!("{}", rustfmt_bindings);
+                eprintln!("{}", rustfmt_stderr);
+            },
+            Ok(None) => (),
+            Err(err) => println!("{:?}", err),
+        }
+        
+        Ok(())
+    }
+    
+    /// Checks if rustfmt_bindings is set and runs rustfmt on the string
+    fn rustfmt_generated_string(&self, source: String) -> io::Result<Option<(String, String)>> {
+        let _t = self.context.timer("rustfmt_generated_string");
+
+        if !self.context.options().rustfmt_bindings {
+            return Ok(None);
+        }
+
+        let rustfmt = if let Ok(rustfmt) = which::which("rustfmt") {
+            rustfmt
+        } else {
+            return Err(io::Error::new(
+                io::ErrorKind::Other,
+                "Rustfmt activated, but it could not be found in global path.",
+            ));
+        };
+        
+        let mut cmd = Command::new(rustfmt);
```

Comment:
This is going to fail with `rustfmt-nightly` which is where `rustfmt` development is happening these days:

```
$ rustfmt --version
rustfmt: error while loading shared libraries: libsyntax-ff7df0d2d58ad322.so: cannot open shared object file: No such file or directory
$ rustup run nightly rustfmt -- --version
0.2.6-nightly ( )
```

We should do something like this:

```rust
let mut cmd = if let Ok(rustup) = which::which("rustup") {
    let mut cmd = Command::new(rustup);
    cmd.args(["run", "nightly", rustfmt, "--"]);
    cmd
} else {
    Command::new(rustfmt)
};
// ... continue configuring `cmd` ...
```